app/views/my_facilities/index.html.erb

```diff
@@ -5,24 +5,73 @@
       <h2>
         Recent updates
       </h2>
-      <h5 class="font-weight-bold mt-3 mb-1">
-        18-Oct-2020
-      </h5>
-      <p>
-        Mobile friendly navigation: tables scroll sideways better, navigation is easier to use.
+      <p class="c-grey-dark">
+        Last updated on Friday 30-Oct-2020. Have Dashboard feedback? Send Claudio an email at claudio@simple.org.
       </p>
+      <h5 class="font-weight-bold mt-2">
```

Comment:
One bit of non-blocking feedback - can you pull out the recent updates to their own partial? Feels like it would be worth having all of them in their own view.

Reply:
@rsanheim will do that in this PR 👍